### PR TITLE
fix: restore service list as flat list

### PR DESCRIPTION
**Pull request by aaron-steinfeld**
## Description
Restore flat service list (not the tree variant) in addition to the new flat endpoint list.

### Testing
Ran unit tests, verified UI locally

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/components/src/search-box/search-box.component.scss [hunk 1]
```diff
@@ -36,7 +36,7 @@
   }
 
   .input {
-    @include font-title;
+    @include body-1-regular($gray-9);
```
Comment (aaron-steinfeld) [inline projects/components/src/search-box/search-box.component.scss:39]: this isn't really part of this change, but noticed it in the UX for this page.

**Comment by aaron-steinfeld** [comment]
After this change: 
## Endpoint from endpoint list:
image

## Endpoint from service's endpoint list:
image

## Services list:
image

## Endpoints list:
image

File: projects/observability/src/pages/apis/endpoints/endpoint-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="vertical-flex-layout">
+      <ht-page-header></ht-page-header>
+      <ht-navigable-dashboard navLocation="${endpointListDashboard.location}"></ht-navigable-dashboard>
+    </div>
+  `
+})
+export class EndpointListComponent {}
```
Comment (skjindal93) [inline projects/observability/src/pages/apis/endpoints/endpoint-list.component.ts:13]: Have we have started calling apis as endpoints in the code as well? Would that cause confusion with existing components for API like `ApiDetailBreadcrumbResolver`?

**Review by skjindal93 — APPROVED** [review]

File: projects/observability/src/pages/apis/endpoints/endpoint-list.component.ts (hunk 2 above)
Comment (aaron-steinfeld) [reply projects/observability/src/pages/apis/endpoints/endpoint-list.component.ts:13]: It's a bit mixed in code. We should get it all consistent, but figured if we're adding new stuff to make it accurate. Another change that should happen is flattening the structure of pages (removing the `/apis` parent dir)

**Review by jake-bassett — APPROVED** [review]

File: src/environments/environment.ts [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 export const environment = {
   production: false,
-  graphqlUri: 'http://localhost:2020/graphql'
+  graphqlUri: 'http://localhost:23431/graphql'
```
[the remark this answers is not in the document]
Comment (anandtiwary) [reply src/environments/environment.ts:8]: I think it should be 2020 only right?

[the remark this answers is not in the document]
Comment (jake-bassett) [reply src/environments/environment.ts:8]: https://github.com/hypertrace/hypertrace-ui/pull/522/files

[the remark this answers is not in the document]
Comment (aaron-steinfeld) [reply src/environments/environment.ts:8]: yeah my bad. this is the HT graphql port (23431) but usually devs are running against a full cluster which puts everything behind 2020.

**Pull request merged**